### PR TITLE
Mark skipdownloaderrors as safe

### LINKED ISSUE #4467: lfs.skipdownloaderrors is not considered a safe key
**Describe the bug**
If you put `lfs.skipdownloaderrors` in `.gitattributes` then Git LFS ignores it with the error

```
WARNING: These unsafe lfsconfig keys were ignored:

  lfs.skipdownloaderrors
```

**To Reproduce**
1. Create a repo using LFS.
2. Add `.gitattributes` with `lfs.skipdownloaderrors` set to `true`.
3. `git push`

**Expected behavior**
It shouldn't print a warning, and when you clone the repo it should respect the setting and ignore download errors.

**System environment**
    git-lfs/2.13.2 (GitHub; darwin amd64; go 1.15.6)

**Output of `git lfs env`**
```
git-lfs/2.13.2 (GitHub; darwin amd64; go 1.15.6)
git version 2.30.0

WARNING: These unsafe lfsconfig keys were ignored:

  lfs.skipdownloaderrors
Endpoint=redacted (auth=none)
  SSH=redacted
Local*=redacted
LocalReferenceDirs=
TempDir=redacted
ConcurrentTransfers=8
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=redacted
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic,lfs-standalone-file
UploadTransfers=basic,lfs-standalone-file
GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh
GIT_EXEC_PATH=/usr/local/Cellar/git/2.30.0/libexec/git-core
git config filter.lfs.process = "git-lfs filter-process"
git config filter.lfs.smudge = "git-lfs smudge -- %f"
git config filter.lfs.clean = "git-lfs clean -- %f"
```

**Additional context**

This issue is basically the same as #2490. It's hard to see how `skipdownloaderrors` could possibly be unsafe so I think it should be whitelisted.

### PR DESCRIPTION
**Pull request by Timmmm**
This option can't be used to do anything dangerous.

Fixes #4467

### CODE REVIEW

File: config/git_fetcher.go [hunk 1]
```diff
@@ -190,4 +190,5 @@ var safeKeys = []string{
 	"lfs.locksverify",
 	"lfs.pushurl",
 	"lfs.url",
+	"lfs.skipdownloaderrors",
```
Comment (chrisd8088) [inline config/git_fetcher.go:193]: One suggestion to keep these in alphabetical order:

```suggestion
	"lfs.skipdownloaderrors",
	"lfs.url",
```

File: docs/man/git-lfs-config.5.ronn [hunk 2]
```diff
@@ -405,6 +405,7 @@ including and limited to:
 - lfs.url
 - remote.{name}.lfsurl
 - lfs.{*}.access
+- lfs.skipdownloaderrors
```
Comment (chrisd8088) [inline docs/man/git-lfs-config.5.ronn:408]: In this case, we might also fix the ordering of `lfs.{*}.access` and `remote.{*}.access` while we're at it.

```suggestion
- lfs.skipdownloaderrors
- lfs.url
- lfs.{*}.access
- remote.{name}.lfsurl
```

**Review by chrisd8088 — CHANGES_REQUESTED** [review]
Hey, thanks for the bug report and patch!

I think this should be OK, but I'm wondering if you would mind reordering the new lines so they fall in alphabetical order in the respective lists, something like the suggestions I added (and in one case, it could help fix another ordering typo in the docs).

Thanks again very much for this PR, and welcome to the project!

**Comment by Timmmm** [comment]
Ooo I can just click buttons - fancy. Thanks for the quick reply!

**Review by chrisd8088 — APPROVED** [review]
Thanks very much, this looks good to me!  Thanks again for the bug report and patch; it's much appreciated.

**Pull request merged**